package.json

```
@@ -15,11 +15,10 @@
     "cy:open": "cypress open",
     "cy:run": "cypress run",
     "cy:test": "cypress run --spec",
-    "deploy-prod:do-not-use-directly": "yarn build:prod && BUCKET=evergreen.spruce yarn deploy; yarn upload-source-maps;",
-    "deploy:beta": ". ./scripts/app-version.sh && yarn build:beta && BUCKET=evergreen-beta.spruce yarn deploy",
-    "deploy:prod": "node scripts/deploy-prod.js",
-    "deploy:staging": ". ./scripts/app-version.sh && yarn build:staging && BUCKET=evergreen-staging.spruce yarn deploy",
-    "deploy": "aws s3 sync build/ s3://${BUCKET}/ --acl public-read --follow-symlinks --delete --exclude env/.cmdrc.json",
+    "deploy:beta": "yarn build:beta && env-cmd -e beta  env/.cmdrc.json yarn deploy:do-not-use",
+    "deploy:staging": "yarn build:staging && env-cmd -e staging -r env/.cmdrc.json yarn deploy:do-not-use",
```

Potentially silly question but why does one have `-r` and the other doesn't?
```suggestion
    "deploy:beta": "yarn build:beta && env-cmd -e beta -r env/.cmdrc.json yarn deploy:do-not-use",
    "deploy:staging": "yarn build:staging && env-cmd -e staging -r env/.cmdrc.json yarn deploy:do-not-use",
```

---

Great catch!